pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -150,7 +150,7 @@ buildPythonPackage rec {
   # ];
 
   # Ultimately, torch is our test suite:
-  passthru.tests = { inherit torchWithRocm; };
+  passthru.tests = { inherit torch; };
```

Comment:
Perhaps you should split this and the the xformers change into another pr.

Reply:
Split xformers update, also added cuda support:
https://github.com/NixOS/nixpkgs/pull/285495